### PR TITLE
[WIP] - Enhancement/toml bump version to 0.9.0

### PR DESCRIPTION
**Pull request by efenfauzi**
0.9.0 (2020-05-19)
==================
- [x] Add multiple region login
- [x] Fix login session on windows

### CODE REVIEW

File: neo/libs/login.py [hunk 1]
```diff
@@ -26,7 +29,9 @@ def get_password():
 
 
 def get_region():
+    get_region_list()
```
Comment (azzamsa) [inline neo/libs/login.py:32]: didn't return anything? I thinks it's better with `show` rather than `get`

File: neo/libs/login.py [hunk 2]
```diff
@@ -37,6 +42,31 @@ def get_region():
...
+            [[key, value] for key, value in GLOBAL_REGION.items()],
+            headers=["Region", "Auth URL"],
+            tablefmt="fancy_grid",
+        )
+    )
+
+
+def get_region_toml(username, password, auth_url):
+    config = ""
+    for key, value in GLOBAL_REGION.items():
```
Comment (azzamsa) [inline neo/libs/login.py:57]: it's rare to use `something.items()` then loop the key and value over it. But if there is no  other way, it's okay.

File: neo/libs/login.py [hunk 3]
```diff
@@ -37,6 +42,31 @@ def get_region():
...
+        status = "ACTIVE" if value == auth_url else "IDLE"
+        project_id = get_project_id(username, password, value, GLOBAL_USER_DOMAIN_NAME)
+        config += "\n"
+        config += "[region.{}]\n".format(key)
```
Comment (azzamsa) [inline neo/libs/login.py:61]: [OPT] you can use f-string for better readability

File: neo/libs/login.py [hunk 4]
```diff
@@ -48,22 +78,34 @@ def generate_session(username, password, auth_url, user_domain_name, project_id=
...
+    return os.path.join(GLOBAL_HOME, ".neo", "config.toml")
 
 
-def create_env_file(username, password, project_id, auth_url, user_domain_name):
+def check_env():
+    return os.path.isfile(get_toml_config())
+
+
+def create_env_file(username, password, auth_url):
+    config_list = """
```
Comment (azzamsa) [inline neo/libs/login.py:93]: IMHO it will be more readable to use f-string than string substitution

File: neo/libs/login.py [hunk 5]
```diff
@@ -241,39 +284,48 @@ def login_check(username=None, region=None):
...
         login_check(username, region)
 
 
 def do_logout():
+    temp = utils.tmp_dir()
     if check_session():
         home = os.path.expanduser("~")
-        os.remove("/tmp/session.pkl")
-        os.remove(home + "/.neo.env")
+        os.remove("{}/session.pkl".format(temp))
```
Comment (azzamsa) [inline neo/libs/login.py:308]: windows support?

Comment (efenfauzi) [reply neo/libs/login.py:308]: after os removing method, execute delete dir method. anyway, i will check this to ensure it.

File: neo/libs/login.py (hunk 1 above)
Comment (efenfauzi) [reply neo/libs/login.py:32]: oks...noted corresponding method is **show**

File: neo/libs/login.py (hunk 5 above)
Comment (azzamsa) [reply neo/libs/login.py:308]: there is a forward slash `/`, and windows use backward slash. that's the hint

File: neo/libs/login.py (hunk 2 above)
Comment (efenfauzi) [reply neo/libs/login.py:57]: GLOBAL_REGION type is dict and define as global variabel. will check later to use another method

File: neo/libs/login.py (hunk 3 above)
Comment (efenfauzi) [reply neo/libs/login.py:61]: oks... thanks for suggestion...

File: neo/libs/login.py (hunk 4 above)
Comment (efenfauzi) [reply neo/libs/login.py:93]: same as previous... thanks for sugestions. will checks for another method.

**Pull request merged**